draft-ietf-mimi-content.md

```diff
@@ -1184,6 +1212,26 @@ Clients compliant with this specification must be able to download
 ExternalParts with `http` and `https` URLs, and decrypt downloaded content
 encrypted with the AES-128-GCM AEAD algorithm.
 
+### Specifics of Github Flavored Markdown in MIMI
+
+A MIMI content client supports GitHub Flavored Markdown as defined in [@!GFM],
+with one exception (no raw HTML support) and a fixed list of supported
+extensions, as further described in the bullet points below.
```

Comment:
I did not know that GFM is a strict superset of Commonmark. GFM = CommonMark + tables + task lists + strikethrough + more autolinks + less raw html. Extensions are not optional, so actually my comment about deciding on extensions did not make sense.